packages/queries/src/query.ts

```
@@ -343,172 +195,97 @@
+        include: TInclude,
+    ): QueryBuilder<DB, TTable, AppendToSchema<TResultSchema, TInclude>> {
+        const query = this.query;
+
+        const schema = t.Composite([
+            this.schema,
+            t.Object(
+                Object.entries(include)
+                    .map(([key, value]) => {
+                        return { [key]: value.schema };
+                    })
+                    .reduce((acc, curr) => ({ ...acc, ...curr }), {}) as any,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)